examples/basic_candlepin.pp

```diff
@@ -27,9 +29,9 @@
   notify  => Service['tomcat'],
 } ->
 package { ['java']: } ->
-exec { 'add ca cert':
-  command => "/usr/bin/keytool -import -trustcacerts -v -keystore /etc/candlepin/certs/keystore -storepass ${keystore_password} -alias candlepin-ca -file /etc/candlepin/certs/candlepin-ca.crt -noprompt",
-  unless  => "/usr/bin/keytool -list -keystore ${keystore} -storepass ${keystore_password} -alias candlepin-ca | grep $(openssl x509 -noout -fingerprint -in ${ca_cert} | cut -d '=' -f 2)",
+exec { 'Create truststore':
+  command => "/usr/bin/keytool -import -v -keystore ${truststore} -alias candlepin-ca -file ${ca_cert} -noprompt -storepass ${truststore_password} -storetype pkcs12",
+  creates => $truststore,
```

Comment:
I'm a bit worried about idempotency - if `$ca_cert` changes, the truststore will not be updated.

Reply:
Should that happen in this test scenario?